plugin-descriptor.properties

```diff
@@ -3,7 +3,7 @@
 description=Provide access control related features for OpenSearch 1.0.0
 #
 # 'version': plugin's version
-version=1.0.1.0
+version=1.1.0.0
```

Comment:
This is security plugin version, not the OpenSearch version. If the goal of this PR is to add `-SNAPSHOT` suffix in the OpenSearch version, this line doesn't need to be changed.

Reply:
The plugin version needs to be incremented because [we're building OpenSearch 1.1](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/opensearch-1.1.0.yml). Every other plugin says it's version 1.1.0.0-SNAPSHOT.

Reply:
This properties file by default has a number without snapshot. Then, with a profile, we change it to be `-SNAPSHOT` unless `-Dbuild.snapshot=false`. You can't have the logical condition the other way around if you want the default to be `-Dbuild.snapshot=false`.